src/components/mediaManager/AposMediaManager.vue

```diff
@@ -238,6 +239,12 @@ export default {
 
 <style lang="scss" scoped>
 @import '../../scss/_mixins';
+
+  .apos-media-manager /deep/ .apos-media-manager-toolbar {
+    position: relative;
+    z-index: 3;
```

Comment:
In core, z-index is handled through global variables. I foresee resolving this floating number there.